### PR TITLE
Smaller fixes

### PR DESCRIPTION
**Pull request by mashehu**
Not sure if you want to actually show that some modules have empty search patterns. I just found it a bit confusing.

### CODE REVIEW

File: src/components/DarkModeToggle.svelte [hunk 1]
```diff
@@ -17,13 +17,6 @@
...
-<button class="rounded-full bg-gray-800 px-2 py-1" on:click={() => toggleTheme()}>
-  <Icon
-    icon="ph:moon-bold"
-    class={"mr-1 inline h-4 w-4 " + (theme === "light" ? "text-gray-600" : "text-gray-300")}
-  />
-  <Icon
-    icon="ph:sun-bold"
-    class={"inline h-4 w-4  " + (theme !== "light" ? "text-gray-600" : "text-gray-300")}
-  />
+<button class="" on:click={() => toggleTheme()}>
```
Comment (ewels) [inline src/components/DarkModeToggle.svelte:20]: Why did you remove the button classes?

File: src/pages/modules/[module].astro [hunk 2]
```diff
@@ -39,7 +39,13 @@ const module_search_patterns = Object.keys(search_patterns)
   <div class="prose max-w-none dark:prose-invert">
     <h2>Description</h2>
     <Content />
-    <h2>File search patterns</h2>
-    <Code code={stringify(module_search_patterns, null, 2)} lang="yaml" />
+    {
+      Object.keys(module_search_patterns).length > 0 && (
```
[the remark this answers is not in the document]
Comment (ewels) [reply src/pages/modules/[module].astro:43]: I need to add a lint test on the MultiQC repo to check that every module has some search patterns (and that there are no search patterns without modules docs).

[the remark this answers is not in the document]
Comment (mashehu) [reply src/pages/modules/[module].astro:43]: jep, but I can't find it now. Maybe it was fixed with the pull of the multiqc repo

[the remark this answers is not in the document]
Comment (ewels) [reply src/pages/modules/[module].astro:43]: I guess it doesn't hurt anyway.. 👍🏻
